app.ts

```diff
@@ -1,13 +1,30 @@
-import { Response, Request } from "express";
+import { Request, Response } from "express";
+import { JobRole } from "./model/jobRole";
+import { Capability } from "./model/capability";
+import { BandLevel } from "./model/bandLevel";
+
 
 const express = require("express");
 const path = require("path");
 const nunjucks = require("nunjucks");
 const session = require("express-session");
-process.env["SESSION_SECRET"] = "your_secret_here";
 
 const app = express();
 
+
+app.use("/public", express.static(path.join(__dirname, "public")));
+ 
+app.use(express.json());
+ 
+app.use(express.urlencoded({ extended: true}));
+
+app.use(session({secret: "NOT HARDCODED SECRET", cookie: {maxAge: 60000}}));
```

Comment:
Is this a hard coded secret ???? : 🤐  -> Would be better to have this as an environment variable.